### DISCUSSION TITLE
Form validation in fluentui/react-northstar

### DISCUSSION THREAD

**Discussion by Sandeep-FED** [microsoft/fluentui#24364] [Q&A] [answered]
Hi guys, I'm trying to build a form using react-northstar components, So I would like to know is there any builtin mechanism in in this library for doing inline form validation, I'm a newbie to react.

Link to the library: https://fluentsite.z22.web.core.windows.net/0.61.0/

Please help me with the form validation.

**Comment by layershifter** [comment, marked as answer]
There is no built-in validation in Fluent UI React Northstar for form components. But, we have a prototype for it using [Formik](https://formik.org/) and [react-hook-form](https://react-hook-form.com/): https://fluentsite.z22.web.core.windows.net/0.64.0/prototype-form-validation

You can find source code there:
- https://github.com/microsoft/fluentui/blob/master/packages/fluentui/react-northstar-prototypes/src/prototypes/FormValidation/Formik.tsx
- https://github.com/microsoft/fluentui/blob/master/packages/fluentui/react-northstar-prototypes/src/prototypes/FormValidation/FormHooks.tsx

**Reply by Sandeep-FED** [reply]
Thank you, I'll check this.
